schema/note.mod

```diff
@@ -1064,39 +1064,40 @@
 <!ELEMENT articulations
 	((accent | strong-accent | staccato | tenuto |
 	  detached-legato | staccatissimo | spiccato |
-	  scoop | plop | doit | falloff | breath-mark | 
-	  caesura | stress | unstress | other-articulation)*)>
+	  scoop | plop | doit | falloff | breath-mark |
+	  caesura | stress | unstress | soft-accent | 
+	  other-articulation)*)>

```

Comment:
Thanks for catching that I omitted the element definition in note.mod. I just pushed new code up that includes this.